src/style/style.js

```diff
@@ -512,9 +512,10 @@ class Style extends Evented {
         this._checkLoaded();
 
         if (this.sourceCaches[id] === undefined) {
-            throw new Error('There no source with this ID');
+            throw new Error('There is no source with this ID');
         }
```

Comment:
Since this is an internal invariant, please replace with `assert(this.sourceCaches[id] === undefined, 'There is no source with this ID')`